src/commons/controlBar/ControlBarShareButton.tsx

```diff
@@ -84,10 +131,13 @@ export class ControlBarShareButton extends React.PureComponent<ControlBarShareBu
               </div>
             )
           ) : (
-            <div key={this.props.shortURL}>
-              <input defaultValue={this.props.shortURL} readOnly={true} ref={this.shareInputElem} />
+            // <div key={this.props.shortURL}>
```

Comment:
If it's not used anymore, old code should be removed.

Reply:
Removed.